src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx

```diff
@@ -65,11 +99,52 @@ export const ChooseRightsPage = () => {
     void dispatch(removeServiceResource(identifier));
   };
 
+  const groupServices = (): DelegationResourceDTO[][] => {
+    return Object.values(
+      selectedRights.reduce((grouped, item) => {
+        const { serviceIdentifier } = item;
+        grouped[serviceIdentifier] = [];
+        grouped[serviceIdentifier].push(item);
+
+        return grouped;
+      }, {}),
+    );
+  };
+
+  const postDelegations = () => {
+    const groupedList = groupServices();
+    setDelegationCount(groupedList.length);
+
+    groupedList.map((item) => {
+      const delegationInput: DelegationInputDto = {
+        // TODO: make adjustments to codeline below when we get GUID from altinn2
+        To: [new IdValuePair('urn:altinn:ssn', '50019992')],
+        Rights: item.map((content) => {
+          return new DelegationRequestDto(
+            // TODO: make adjustments to codline below when we get urn from resourceregistry
+            'urn:altinn:resource',
```

Comment:
That's why I added the TODO 😅
As we've previously [discussed](https://altinndevops.slack.com/archives/CE5E06MC7/p1691494880138869) this will be fixed when the ResourceRegistry adds urn to their response

Reply:
Aha, I didn't realize that was what the comment meant 😅
I thought it just said to switch one urn with another, not to implement different cases for handling when there are more than one urn for a resource.
But if the resource registry change hasn't been rolled out yet, it can probably wait until another task, in which case such a task should be created. (If it isn't already?)

Reply:
Ahh I see! Yeah it's still a little unclear to me how it will become finally, so would like to see the actual change in the response from the API before spending much time trying to foresee how it will become and then maybe it will not work in the end...

I'll make a task about implementing this functionality that we can do when we've seen that they've updated the API